queries.sql

```diff
@@ -6,3 +6,60 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered;
 SELECT * FROM animals WHERE name != 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+----------------------------------------------------------------
+
+BEGIN;
+UPDATE animals
+SET species = 'unspecified';
+
+ROLLBACK;
+
```

Comment:
- [ ]  As requested by the previous reviewer, kindly add the `SELECT * FROM animals;` before and after `ROLLBACK`. This is to verify changes where actually made and rolled back while running the queris.

Reply:
Fixed.